src/DetailsView/components/getting-started-view.tsx

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as styles from 'DetailsView/components/assessment-view.scss';
+import * as React from 'react';
+
+export interface GettingStartedViewProps {
+    gettingStartedContent: JSX.Element;
+}
+
+export class GettingStartedView extends React.Component<GettingStartedViewProps> {
```

Comment:
why not a `NamedFC`? Just asking

Reply:
I was modeling it after the CollapsibleComponent, which is what is currently used for the Getting Started view, but I will change it to `NamedFC`, since this is such a simple component.

Reply:
Thanks for catching this!